src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+    let totalCost = days * dailyRate;
+
+    if (days >= 7) {
+        totalCost -= 50;
+    } else if (days >= 3) {
+        totalCost -= 20;
+    }
```

Comment:
Prefer 'if' with 'return' over 'if else' to simplify the conditions. Since you return 'totalCost' at the end, you can remove the 'else' and have two separate 'if' conditions.